iModelCore/ECDb/ECDb/ECSql/ValueExp.cpp

```diff
@@ -817,47 +817,92 @@ Exp::FinalizeParseStatus MemberFunctionCallExp::_FinalizeParsing(ECSqlParseConte
     if (mode == Exp::FinalizeParseMode::AfterFinalizingChildren)
         {
         if (this->m_tableValuedFunc) {

```

Comment:
You created a IsTableValuedFunc() method for this check. Either stick with it and use it everywhere or do this everywhere.

Reply:
Done